src/goto-diff/unified_diff.cpp

```diff
@@ -33,8 +33,8 @@ void unified_difft::get_diff(
   dest.clear();
 
   differences_mapt::const_iterator entry=
-    differences_map.find(function);
-  if(entry==differences_map.end())
+    differences_map_.find(function);
```

Comment:
As discussed in some other pull request some days ago, I don't know where this need for renaming is coming from _under current coding guidelines_.

Reply:
It's been agreed with @peterschrammel that using an underscore suffix is acceptable when naming private data members. In this particular case, it allows us to name the getter `differences_map()` (which is nicer than `get_differences_map`). If we omitted the suffix, the identifiers would collide.